tests/nginx-tests/tengine-tests/resolver.t

```diff
@@ -230,7 +230,7 @@ EOF
 
 $t->run();
 
-like(http_get_host("/", "www.taobao.com"), qr/HTTP\/1.1 302/, 'resolver_file to resolv4.conf');
+like(http_get_host("/", "www.taobao.com"), qr/HTTP\/1.1 \d\d\d/, 'resolver_file to resolv4.conf');
```

Comment:
www.taobao.com replys 301 currenlty. 

This is not a good test case. It depends on an external uncontrollable backend server.
So I create a new test case in this pr: resolver_file.t.